src/github/issue-comment.ts

```diff
@@ -21,10 +18,7 @@ export const issueCommentWebhookHandler = async (
 	} = context.payload;
 	let linkifiedBody;
 
-	const gitHubBaseUrl = await getGitHubBaseUrl(jiraHost);
-	const githubClient = await gheServerAuthAndConnectFlowFlag(jiraHost)
-		? new GitHubInstallationClient(getCloudInstallationId(githubInstallationId, gitHubBaseUrl), context.log, gitHubBaseUrl)
-		: new GitHubInstallationClient(getCloudInstallationId(githubInstallationId), context.log);
+	const githubClient = await createInstallationClient(githubInstallationId, jiraHost, context.log);
```

Comment:
Consolidated the creation of an installation client in `createInstallationClient` (I didn't replace all occurrences, this is just an example)